docs/compression_algorithms/Quantization.md

```diff
@@ -305,6 +305,7 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
 ```
 {
     "algorithm": "quantization",
+    "preset": "mixed", // The preset defines the quantization schema for weights and activations. The parameter takes values 'permormance' or 'mixed'. The mode 'performance' defines symmetric weights and activations. The mode 'mixed' defines symmetric 'weights' and asymmetric 'activations'. Weights quantize with 'per_channel': True by default. Any preset parameters can be overriden by sections 'weights' and 'activations'.
```

Comment:
That is the default preset? I suggest using "performance" and specify it in the document. I mean replace: ` "preset": "mixed"` with ` "preset": "performance"`

Reply:
`overriden` -> `overridden`. I remember we decided to throw an exception if both preset and manual settings are provided.

Reply:
@AlexKoff88 After additional discussion with @alexsu52 , we decided to make a different behavior. Please see #860

Reply:
@asenina, can you explain what do you mean by a different behavior?